### PR TITLE
Follow-up on the previous PR: make large numbers human-readable.

### PR DESCRIPTION
**Pull request by mavann**
Make numbers on the Y-axis human-readable.

### CODE REVIEW

File: Arrt/View/ModelEditor/Stats/StatsGraph.cpp [hunk 1]
```diff
@@ -55,7 +55,20 @@ namespace
 
     QString toString(double d)
     {
-        return DoubleFormatter::toString(d, "%.2f", true);
+        // Print large numbers in a humand-friendly way. We add a space
+        // at the end so numbers with units don't look weird, e.g. "1M ms"
+        QString suffix;
+        if (d > 1000000)
```
Comment (mafranc) [inline Arrt/View/ModelEditor/Stats/StatsGraph.cpp:61]: You can also just check it's divisible by 1000000 (or 1000), which should be the case for how the graph is computed

File: ArrtModel/Model/ArrServiceStats.cpp [hunk 2]
```diff
@@ -58,7 +58,7 @@ void ArrServiceStats::updateStats(RR::ApiHandle<RR::AzureSession> session)
             m_currentStats.m_memoryCPU.addValue(m_lastPerformanceAssessment.memoryCPU.aggregate, m_tick);
             m_currentStats.m_memoryGPU.addValue(m_lastPerformanceAssessment.memoryGPU.aggregate, m_tick);
             m_currentStats.m_networkLatency.addValue(m_lastPerformanceAssessment.networkLatency.aggregate, m_tick);
-            m_currentStats.m_polygonsRendered.addValue(m_lastPerformanceAssessment.networkLatency.aggregate, m_tick);
+            m_currentStats.m_polygonsRendered.addValue(m_lastPerformanceAssessment.polygonsRendered.aggregate, m_tick);
```
Comment (mafranc) [inline ArrtModel/Model/ArrServiceStats.cpp:61]: isn't this already merged?

Comment (mavann) [reply ArrtModel/Model/ArrServiceStats.cpp:61]: Oh yeah forgot to pull :)

File: Arrt/View/ModelEditor/Stats/StatsGraph.cpp (hunk 1 above)
Comment (mavann) [reply Arrt/View/ModelEditor/Stats/StatsGraph.cpp:62]: I've added two functions and made it clearer what it's used for.

File: Arrt/View/ModelEditor/Stats/StatsGraph.cpp [hunk 3]
```diff
@@ -390,7 +408,7 @@ void StatsGraph::setHighlightX(std::optional<float> x)
                             m_highlightedPoints.clear();
                             m_highlightedX = chosen.x();
                         }
-                        m_highlightedPoints.push_back({m_infos[valIdx].m_color, QString("%1: %2%3").arg(m_infos[valIdx].m_name).arg(toString(chosen.y())).arg(m_infos[valIdx].m_units), chosen});
+                        m_highlightedPoints.push_back({m_infos[valIdx].m_color, QString("%1: %2%3").arg(m_infos[valIdx].m_name).arg(toHumanReadableString(chosen.y())).arg(m_infos[valIdx].m_units), chosen});
```
Comment (mafranc) [inline Arrt/View/ModelEditor/Stats/StatsGraph.cpp:411]: I wouldn't simplify the highlighted values on the tooltip (so we don't have approximation). I would just simplify the Y labels

Comment (mavann) [reply Arrt/View/ModelEditor/Stats/StatsGraph.cpp:411]: Discussed offline, it looks a lot neater to have the simplified string in the tooltip too.

File: Arrt/ViewUtils/Formatter.h [hunk 4]
```diff
@@ -92,6 +92,26 @@ class DoubleFormatter
...
+    // bigger than 1,000 and 1,000,000.
+    static QString toHumanReadableString(double value)
+    {
+        // Print large numbers in a humand-friendly way. We add a space
+        // at the end so numbers with units don't look weird, e.g. "1M ms"
+        QString suffix;
+        if (value > 1000000)
+        {
+            value /= 1000000;
+            suffix = "M ";
```
Comment (mafranc) [inline Arrt/ViewUtils/Formatter.h:105]: You shouldn't add the space here

**Review by mafranc — APPROVED** [review]

File: Arrt/ViewUtils/Formatter.h (hunk 4 above)
Comment (mavann) [reply Arrt/ViewUtils/Formatter.h:105]: Fixed.

**Pull request merged**